src/main_helpers.js

```diff
@@ -87,3 +89,58 @@ export function createPythonFlaskProcess(investExe) {
     logger.error('no existing invest installations found');
   }
 }
+
+export function extractZipInplace(zipFilePath) {
+  return new Promise((resolve, reject) => {
+    const extractToDir = path.dirname(zipFilePath);
+    yauzl.open(zipFilePath, { lazyEntries: true }, (err, zipfile) => {
+      if (err) throw err;
+      zipfile.readEntry();
+      zipfile.on('entry', (entry) => {
+        if (/\/$/.test(entry.fileName)) {
+          // if entry is a directory
+          fs.mkdir(path.join(extractToDir, entry.fileName), (err) => {
+            if (err) {
+              if (err.code === 'EEXIST') { } else logger.error(err);
+            }
+            zipfile.readEntry();
+          });
+        } else {
+          console.log(entry.fileName);
```

Comment:
Is this `console.log` left here intentionally? Just checking.

Reply:
Nope, thanks!